enterprise/reporting/reporting.go

```diff
@@ -374,6 +384,23 @@ func (r *HandleT) mainLoop(ctx context.Context, clientName string) {
 	r.getMinReportedAtQueryTime = stats.Default.NewTaggedStat(StatReportingGetMinReportedAtQueryTime, stats.TimerType, tags)
 	r.getReportsQueryTime = stats.Default.NewTaggedStat(StatReportingGetReportsQueryTime, stats.TimerType, tags)
 	r.requestLatency = stats.Default.NewTaggedStat(StatReportingHttpReqLatency, stats.TimerType, tags)
+
+	lastReportedAt := time.Now().UTC().Unix() / 60
+	go func() {
+		// for montering reports pileups
+		for {
+			currentMs := time.Now().UTC().Unix() / 60
+			stats.Default.NewTaggedStat(
+				"reporting_metrics_ lag_minutes", stats.GaugeType, stats.Tags{"client": clientName},
```

Comment:
```suggestion
			stats.Default.NewTaggedStat(
				"reporting_metrics_lag_minutes", stats.GaugeType, stats.Tags{"client": clientName},
```
Any reason why we have an extra space in the tag name?

Reply:
removed extra space.